docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 Sphinx
+alabaster
```

Comment:
Please also add `sphinx-design`.

Reply:
Nice catch. `alabaster` is not needed actually, it probably got mixed up while looking proper package.

I was under the assumption the CI does build the docs.